renku/api/storage.py

```diff
@@ -132,12 +132,13 @@ def track_paths_in_storage(self, *paths):
                 # TODO create configurable filter and follow .gitattributes
                 track_paths.append(str(path))
 
-        call(
-            self._CMD_STORAGE_TRACK + track_paths,
-            stdout=PIPE,
-            stderr=STDOUT,
-            cwd=str(self.path),
-        )
+        if any(track_paths):
```

Comment:
why do you need the `any`?

Reply:
Can we clean it up if we don't need it? I don't think we need it since it will always contain Path objects or an exception will be raised instead of None being put into a list.